<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenInstrumentClass</class>
 <widget class="QMainWindow" name="OpenInstrumentClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>984</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OpenInstrument</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>984</width>
     <height>38</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionRun"/>
   </widget>
   <widget class="QMenu" name="menuAnalyze">
    <property name="title">
     <string>Analyze</string>
    </property>
    <addaction name="actionAnalyze_Data"/>
    <addaction name="separator"/>
    <addaction name="actionData_Processing"/>
    <addaction name="actionRegression"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionEdit_Tasks"/>
    <addaction name="actionEdit_Sequence"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_OpenInstrument"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuAnalyze"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionEdit_Sequence">
   <property name="icon">
    <iconset>
     <normaloff>icons/sequence.png</normaloff>icons/sequence.png</iconset>
   </property>
   <property name="text">
    <string>Edit Sequence</string>
   </property>
  </action>
  <action name="actionEdit_Tasks">
   <property name="icon">
    <iconset>
     <normaloff>icons/task.png</normaloff>icons/task.png</iconset>
   </property>
   <property name="text">
    <string>Edit Tasks</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="OpenInstrument.qrc">
     <normaloff>:/OpenInstrument/icons/open folder.png</normaloff>:/OpenInstrument/icons/open folder.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="OpenInstrument.qrc">
     <normaloff>:/OpenInstrument/icons/save.png</normaloff>:/OpenInstrument/icons/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionAbout_OpenInstrument">
   <property name="text">
    <string>About OpenInstrument</string>
   </property>
  </action>
  <action name="actionData_Processing">
   <property name="text">
    <string>Data Processing</string>
   </property>
  </action>
  <action name="actionRegression">
   <property name="text">
    <string>Regression</string>
   </property>
  </action>
  <action name="actionAnalyze_Data">
   <property name="icon">
    <iconset>
     <normaloff>icons/analyze.png</normaloff>icons/analyze.png</iconset>
   </property>
   <property name="text">
    <string>Analyze Data</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="icon">
    <iconset resource="OpenInstrument.qrc">
     <normaloff>:/OpenInstrument/icons/run.png</normaloff>:/OpenInstrument/icons/run.png</iconset>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="OpenInstrument.qrc"/>
 </resources>
 <connections/>
</ui>
